contracts/eip20/EIP20.sol

```diff
@@ -72,7 +76,4 @@ contract EIP20 is EIP20Interface {
     view public returns (uint256 remaining) {
       return allowed[_owner][_spender];
     }
-
```

Comment:
@skmgoldin @simondlr 

I agree, it does seem a bit odd to have these mappings at the bottom. Is there a reason for it?